test/integration/pages/login.py

```diff
@@ -0,0 +1,37 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+"""Login page locators and functions."""
+
+from pypom import Page
+from selenium.webdriver.common.by import By
+
+from pages.home import Home
+
+
+class Login(Page):
+    """Set up the login page locators and functions."""
+
+    _password_locator = (By.NAME, 'password')
+    _confirm_password_locator = (By.NAME, 'confirmPassword')
+    _continue_locator = (By.CSS_SELECTOR, 'button.button__button___267m4')
+
+    def wait_for_page_to_load(self):
+        """Page load wait."""
+        self.wait.until(
+            lambda s: any(s.find_elements(*self._password_locator)))
```

Comment:
I think this was my code, and I think we can remove the `any` here, because an empty list is falsey.